pkg/collector/corechecks/ebpf/probe/tcp_queue_length_test.go

```diff
@@ -25,14 +25,20 @@ import (
 
 var (
 	missingBTFS = map[string]struct{}{
-		"4.18.0-1018-azure":            {},
-		"4.18.0-147.43.1.el8_1.x86_64": {},
+		"4.14.275-207.503.amzn2.aarch64": {},
```

Comment:
This is in the archive, so something else must be going on: https://github.com/aquasecurity/btfhub-archive/blob/main/amzn/2/arm64/4.14.275-207.503.amzn2.aarch64.btf.tar.xz